config.py

```diff
@@ -92,6 +92,80 @@ def load(self):
 The head of the HAProxy config. This contains global settings
 and defaults.
 '''))
+        self.add_template(
```

Comment:
The problem i see here with enabling the module all the time is:

-  Lets say initially the `haproxy-map` flag is disabled, you run marathon-lb and the lua module tries reading a non-existent file, so it returns empty response.
-  Now lets run marathon-lb with `haproxy-map` flag, so the map file get created and everything works as expected, the endpoint returns the map.
-  Run marathon-lb again without `haproxy-map` flag, the module looks for the file, which it finds(since it was created in step 2)  and it serves the map file even though the current config is not using the map.